EIPS/eip-6808.md

```diff
@@ -0,0 +1,680 @@
+```
+
+#### `getBindings` function
+
+The function returns the `keyWallets` for the `_account` in a `struct` format.
+
+```solidity
+struct AccountHolderBindings {
+    address firstWallet;
+    address secondWallet;
+}
+```
+
+```solidity
+function getBindings(address _account) external view returns (AccountHolderBindings memory)
+```
+
+#### `resetBindings` function
+
+**Note:** This function is helpful when one of the two `keyWallets` is compromised.
+
+Called from a `keyWallet`, the function resets the `keyWallets` for the `holder` account. MUST fire the `AccountResetBinding` event.
+
+The function SHOULD `throw` if the sender is not a `keyWallet`.
+
+```solidity
+function resetBindings() external returns (bool)
+```
+
+#### `safeFallback` function
+
+**Note:** This function is helpful when the `holder` account is compromised.
+
+Called from a `keyWallet`, this function transfers all the tokens from the `holder` account to the other `keyWallet` and MUST fire the `SafeFallbackActivated` event.
+
+The function SHOULD `throw` if the sender is not a `keyWallet`.
+
+```solidity
+function safeFallback() external returns (bool);
+```
+
+#### `allowTransfer` function
+
+Called from a `keyWallet`, this function is called before a `transfer` function is called.
+
+It allows to transfer a maximum amount, for a specific time frame, to a specific address.
+
+If the amount is 0 then there will be no restriction on the amount.
+If the time is 0 then there will be no restriction on the time.
+If the to address is zero address then there will be no restriction on the to address.
+Or if `_allFunds` is `true`, regardless of the other params, it allows all funds, whenever, to anyone to be transferred.
+
+The function MUST fire `AccountEnabledTransfer` event.
+
+The function SHOULD `throw` if the sender is not a `keyWallet` or if the `_amount` is greater than the `holder` account balance.
+
+```solidity
+function allowTransfer(uint256 _amount, uint256 _time, address _to, bool _allFunds) external returns (bool);
+```
+
+#### `getTransferableFunds` function
+
+The function returns the transfer conditions for the `_account` in a `struct` format.
+
+```solidity
+struct TransferConditions {
+    uint256 amount;
+    uint256 time;
+    address to;
+    bool allFunds;
+}
+```
+
+```solidity
+function getTransferableFunds(address _account) external view returns (TransferConditions memory);
+```
+
+#### `allowApproval` function
+
+Called from a `keyWallet`, this function is called before one of the `approve`, `increaseAllowance` or `decreaseAllowance` function are called.
+
+It allows the `holder` for a specific amount of `_time` to do an `approve`, `increaseAllowance` or `decreaseAllowance` and limit the number of transfers the spender is allowed to do through `_numberOfTransfers` (0 - unlimited number of transfers in the allowance limit).
+
+The function MUST fire `AccountEnabledApproval` event.
+
+The function SHOULD `throw` if the sender is not a `keyWallet`.
```

Comment:
I think the generally accepted term is "revert" (not "throw") but I may be mistaken. I don't Solidity much.